RobotCar/Functions/Pinmap.c

```diff
@@ -0,0 +1,36 @@
+/*
+ * Pinmap.c
+ *
+ * Created: 5-12-2023 12:01:32
+ *  Author: jspaa
+ */ 
+
+#include "Pinmap.h"
+
+void registers_init(void)
```

Comment:
kan beter IO_init(void) zijn

Reply:
gezien hier alleen de IO register worden geinit
```suggestion
void IO_init(void)
```